api/routers/legacy.py

```diff
@@ -259,17 +260,17 @@ async def sql_update_player(player: dict):
     return data.rows2dict() if data is not None else {}
 
 
-async def sql_get_latest_xp_gain(player_id: int):
+async def sql_get_latest_xp_gain(player_ids: List[int]):
```

Comment:
It's because of the duplicates. Whenever I was running get_player I was getting the I'd for Seltzer_Bro and not Seltzer Bro (which is the id hiscores and stuff go in with). I figured if someone has duplicate entries we can just combine them for right now.

Reply:
yes, but you end up getting the first element of the list anyway, or end up double counting the xp gains